specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/subAssessments.json

```diff
@@ -535,6 +535,7 @@
       "in": "path",
       "required": true,
       "type": "string",
+      "pattern": "^[-\\w\\._\\(\\)]+$",
```

Comment:
for the 'AssessmentName' global model to be correct, the same model needs to be used - this change triggered the [OBJECT_ADDITIONAL_PROPERTIES](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md#OBJECT_ADDITIONAL_PROPERTIES) which is benign and won't change (I've checked against the service and the actual payload returned by it is too large and dynamic to add)